lib/views/item/item-detail.js

```diff
@@ -95,15 +114,15 @@ class ItemDetail extends React.Component {
     );
   }
 
-  renderUrlBar({ url }) {
+  renderUrlBar({ url } = {}) {
```

Comment:
Why is this required? You're already passing an empty object if metadata is falsey.